### PR TITLE
DOCUMENTATION: Update Docs Move to STX, Grammar Editing

### PR DESCRIPTION
**Pull request by glhays**
Closes: #112

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,8 +1,8 @@
 # ADotNet
 
-![ADotNet](https://raw.githubusercontent.com/hassanhabib/AdotNet/master/ADotNet/ADotNet_git_logo.png)
+![ADotNet](https://raw.githubusercontent.com/The-Standard-Organization/ADotNet/main/ADotNet/ADotNet_git_logo.png)
 
-[![.NET](https://github.com/hassanhabib/ADotNet/actions/workflows/dotnet.yml/badge.svg)](https://github.com/hassanhabib/ADotNet/actions/workflows/dotnet.yml)
+[![Build](https://github.com/The-Standard-Organization/STX.REST.RESTFulSense.Clients/actions/workflows/build.yml/badge.svg)](https://github.com/The-Standard-Organization/STX.REST.RESTFulSense.Clients/actions/workflows/build.yml)
```
Comment (cjdutoit) [inline README.md:5]: Needs to be ADotNet

**Review by cjdutoit — DISMISSED** [review]
Just the one link that needs to change to ADotNet

**Comment by glhays** [comment]
@Christo FYI this needs reviewed ^^^^

**Review by glhays — COMMENTED** [review]
@cjdutoit This still needs your review, the merge will fix build badge.

**Pull request closed without merging**